bench/real_client.rb

```diff
@@ -0,0 +1,35 @@
+require "nats/io/client"
+require "thread"
+require "benchmark/ips"
+require "protobuf"
+require 'protobuf/message'
+require 'protobuf/rpc/service'
```

Comment:
yeah.. we can totally remove duplication here. and good idea about setting vars. I'll make that change.

1k req/sec is reasonable for a single thread. I have a collection of stats. Let me open an issue and I'll link it here. Here's some relevant comparisons:

ZMQ
```Calculating -------------------------------------
single threaded performance
                        919.255  (±19.6%) i/s -      8.478k in  10.227414s
```

Socket:
```Calculating -------------------------------------
single threaded performance
                        825.149  (± 8.5%) i/s -      8.240k in  10.107899s
```